### DISCUSSION TITLE
How to install the running environment

### DISCUSSION THREAD

**Discussion by sktsherlock** [microsoft/ClimaX#16] [Q&A]
Are there any commands to install the environment that you can refer to?

**Comment by rejuvyesh** [comment]
Assuming you have `conda` installed, you can follow the instructions in https://microsoft.github.io/ClimaX/install/ . You can install `conda` itself by following the instructions in https://docs.anaconda.com/free/anaconda/install/.

**Comment by Ericc1027** [comment]
What can I do? had try many times:Collecting package metadata (repodata.json): done
Solving environment: failed

PackagesNotFoundError: The following packages are not available from current channels:

  - zstd==1.5.2=ha4553b6_0
  - zlib==1.2.13=h166bdaf_4
  - yaml==0.2.5=h7f98852_2
  - xz==5.2.10=h5eee18b_1
  - urllib3==1.26.14=py38h06a4308_0
  - typing_extensions==4.4.0=py38h06a4308_0
  - tornado==6.2=py38h0a891b7_1
  - torchvision==0.13.1=py38_cu113
  - torchaudio==0.12.1=py38_cu113
  - tk==8.6.12=h1ccaba5_0
  - sqlite==3.40.1=h5082296_0
  - shapely==1.8.5=py38hafd38ec_2
  - setuptools==65.6.3=py38h06a4308_0
  - scipy==1.10.0=py38h14f4228_0
  - requests==2.28.1=py38h06a4308_0
  - readline==8.2=h5eee18b_0
  - pyyaml==6.0=py38h0a891b7_5
  - pytorch==1.12.1=py3.8_cuda11.3_cudnn8.3.2_0
  - python==3.8.16=h7a1cb2a_2
  - pysocks==1.7.1=py38h06a4308_0
  - psutil==5.9.4=py38h0a891b7_0
  - portalocker==2.3.0=py38h06a4308_0
  - pip==22.3.1=py38h06a4308_0
  - pillow==9.3.0=py38h6a678d5_2
  - pandas==1.5.3=py38hdc8b05c_0
  - openssl==1.1.1s=h0b41bf4_1
  - openh264==2.1.1=h4ff587b_0
  - numpy-base==1.23.5=py38h31eccc5_0
  - numpy==1.23.5=py38h14f4228_0
  - numcodecs==0.11.0=py38h8dc9893_1
  - numba==0.56.4=py38h9a4aae9_0
  - nettle==3.7.3=hbbd107a_1
  - netcdf4==1.6.2=nompi_py38h2250339_100
  - netcdf-fortran==4.6.0=nompi_he1eeb6f_102
  - ncurses==6.4=h6a678d5_0
  - msgpack-python==1.0.4=py38h43d8883_1
  - mkl_random==1.2.2=py38h51133e4_0
  - mkl_fft==1.3.1=py38hd3c417c_0
  - mkl-service==2.4.0=py38h7f8727e_0
  - mkl==2021.4.0=h06a4308_640
  - markupsafe==2.1.2=py38h1de0b5d_0
  - lz4-c==1.9.4=h6a678d5_0
  - lz4==4.2.0=py38hd012fdc_0
  - llvmlite==0.39.1=py38h38d86a4_1
  - llvm-openmp==15.0.7=h0cdce71_0
  - libzlib==1.2.13=h166bdaf_4
  - libzip==1.9.2=hc869a4a_1
  - libwebp-base==1.2.4=h5eee18b_0
  - libwebp==1.2.4=h11a3e52_0
  - libunistring==0.9.10=h27cfd23_0
  - libtiff==4.5.0=h6a678d5_1
  - libtasn1==4.16.0=h27cfd23_0
  - libstdcxx-ng==12.2.0=h46fd767_19
  - libssh2==1.10.0=haa6b8db_3
  - libpng==1.6.37=hbc83047_0
  - libnghttp2==1.51.0=hdcd2b5c_0
  - libnetcdf==4.8.1=nompi_h21705cb_104
  - libllvm11==11.1.0=he0ac6c6_5
  - libidn2==2.3.2=h7f8727e_0
  - libiconv==1.16=h7f8727e_2
  - libgfortran5==12.2.0=h337968e_19
  - libgfortran-ng==12.2.0=h69a702a_19
  - libgcc-ng==12.2.0=h65d4601_19
  - libffi==3.4.2=h6a678d5_6
  - libev==4.33=h516909a_1
  - libedit==3.1.20191231=he28a2e2_2
  - libdeflate==1.8=h7f8727e_5
  - libcurl==7.87.0=h6312ad2_0
  - libaec==1.0.6=hcb278e6_1
  - lerc==3.0=h295c915_0
  - ld_impl_linux-64==2.38=h1181459_1
  - lcms2==2.12=h3be6417_0
  - lame==3.100=h7b6447c_0
  - krb5==1.20.1=hf9c8cef_0
  - keyutils==1.6.1=h166bdaf_0
  - jpeg==9e=h7f8727e_0
  - intel-openmp==2021.4.0=h06a4308_3561
  - idna==3.4=py38h06a4308_0
  - hdf5==1.12.2=nompi_h2386368_101
  - hdf4==4.2.15=h9772cbc_5
  - gnutls==3.6.15=he1e5248_0
  - gmp==6.2.1=h295c915_3
  - giflib==5.2.1=h5eee18b_1
  - geos==3.11.1=h27087fc_0
  - freetype==2.12.1=h4a9f257_0
  - fftw==3.3.10=nompi_hf0379b8_106
  - ffmpeg==4.3=hf484d3e_0
  - esmpy==8.4.0=nompi_py38h2b78397_1
  - esmf==8.4.0=nompi_hdb2cfa9_3
  - cytoolz==0.12.0=py38h0a891b7_1
  - curl==7.87.0=h6312ad2_0
  - cudatoolkit==11.3.1=h2bc3f7f_2
  - cryptography==38.0.4=py38h9ce1e76_0
  - cftime==1.6.2=py38h26c90d9_1
  - cffi==1.15.1=py38h5eee18b_3
  - ca-certificates==2022.12.7=ha878542_0
  - c-ares==1.18.1=h7f98852_0
  - bzip2==1.0.8=h7b6447c_0
  - brotlipy==0.7.0=py38h27cfd23_1003
  - bottleneck==1.3.6=py38h7e4f40d_0
  - _openmp_mutex==4.5=2_kmp_llvm
  - _libgcc_mutex==0.1=conda_forge

Current channels:

  - https://conda.anaconda.org/pytorch/osx-64
  - https://conda.anaconda.org/conda-forge/osx-64
  - https://repo.anaconda.com/pkgs/main/osx-64
  - https://repo.anaconda.com/pkgs/r/osx-64

To search for alternate channels that may provide the conda package you're
looking for, navigate to

    https://anaconda.org

and use the search bar at the top of the page.

**Reply by rejuvyesh** [reply]
The current conda environment is linux only unfortunately. It seems you are running on osx.
